openwisp_monitoring/device/api/views.py

```diff
@@ -191,15 +191,26 @@ def _write(self, request, pk):
         ct = ContentType.objects.get_for_model(Device)
         for interface in data.get('interfaces', []):
             ifname = interface['name']
+            extra_values = {}
+            field = None
             for key, value in interface.get('statistics', {}).items():
```

Comment:
I thought of doing so but didn't do so due to `statistics_stored`, it can have more fields in the future, that's what I assumed but I can proceed now :smiley:

Reply:
yes that was the initial idea but turned out to be the usual brilliant idea that never materializes and only adds unneeded complexity (yes, I also do this mistake sometimes) so we can remove it :+1: